hw/ip_templates/otp_ctrl/dv/env/seq_lib/otp_ctrl_zeroization_vseq.sv.tpl

```diff
@@ -0,0 +1,276 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Copyright zeroRISC Inc.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+// smoke test vseq to walk through DAI states and request keys
+<%
+from topgen.lib import Name
+
+parts_without_lc = [part for part in otp_mmap["partitions"] if
+                    part["variant"] in ["Buffered", "Unbuffered"]]
+read_locked_csr_parts = [part for part in otp_mmap["partitions"] if
+                         part["read_lock"] == "CSR"]
+write_locked_digest_parts = [part for part in otp_mmap["partitions"] if
+                             part["write_lock"] == "Digest"]
+zeroizable_parts = [part for part in otp_mmap["partitions"] if
+                             part["zeroizable"]]
+buf_parts_without_lc = [part for part in otp_mmap["partitions"] if
+                        part["variant"] == "Buffered"]
+secret_parts = [part for part in otp_mmap["partitions"] if
+                part["secret"]]
+## Partitions + LCI + DAI
+num_err_code = len(otp_mmap["partitions"]) + 2
+%>\
+
+// `define PART_CONTENT_RANGE(i) ${"\\"}
+//     {[PART_BASE_ADDRS[``i``]: PART_OTP_SPECIALS_OFFSETS[``i``] - 1]}
+
+// Digest fields can be zeroized, but the zeroization field should be written to only when the
+// partition is designated to be zeroized.
+// The last 8 bytes or any partition is the zeroizible field, hence ignore it when generating legal
```

Comment:
We changed the naming convention: `zeroizable field` -> `zeroization marker`. Also, this is only true for zeroizable partitions, so the content range should depend on PartInfo[i].zeroizable.